includes/class-wc-stripe-order.php

```diff
@@ -0,0 +1,740 @@
+	}
+
+	/**
+	 * Get the refund ID.
+	 *
+	 * @return string
+	 */
+	public function get_refund_id() {
+		return $this->get_meta( '_stripe_refund_id' );
+	}
+
+	/**
+	 * Set the Multibanco data.
+	 *
+	 * @param $data array The Multibanco data to set.
+	 * @return void
+	 */
+	public function set_multibanco_data( $data ) {
+		$this->update_meta_data( '_stripe_multibanco', $data );
+	}
+
+	/**
+	 * Get the Multibanco data.
+	 *
+	 * @return array
+	 */
+	public function get_multibanco_data() {
+		return $this->get_meta( '_stripe_multibanco' );
+	}
+
+	/**
+	 * Set the Stripe intent ID.
+	 *
+	 * @param $intent_id string The intent ID to set.
+	 * @return void
+	 */
+	public function set_intent_id( $intent_id ) {
+		$this->update_meta_data( '_stripe_intent_id', $intent_id );
+	}
+
+	/**
+	 * Get the Stripe intent ID.
+	 *
+	 * @return string
+	 */
+	public function get_intent_id() {
+		return $this->get_meta( '_stripe_intent_id' );
+	}
+
+	/**
+	 * Set the UPE payment type.
+	 *
+	 * @param $payment_type string The payment type to set.
+	 * @return void
+	 */
+	public function set_upe_payment_type( $payment_type ) {
+		$this->update_meta_data( '_stripe_upe_payment_type', $payment_type );
+	}
+
+	/**
+	 * Get the UPE payment type.
+	 *
+	 * @return string
+	 */
+	public function get_upe_payment_type() {
+		return $this->get_meta( '_stripe_upe_payment_type' );
+	}
+
+	/**
+	 * Set the Stripe source ID.
+	 *
+	 * @param $source_id string The Stripe source ID.
+	 * @return void
+	 */
+	public function set_source_id( $source_id ) {
+		$this->update_meta_data( '_stripe_source_id', $source_id );
+	}
+
+	/**
+	 * Get the Stripe source ID.
+	 *
+	 * @return string
+	 */
+	public function get_source_id() {
+		return $this->get_meta( '_stripe_source_id' );
+	}
+
+	/**
+	 * Set the Stripe customer ID.
+	 *
+	 * @param $customer_id string The Stripe customer ID.
+	 * @return void
+	 */
+	public function set_stripe_customer_id( $customer_id ) {
+		$this->update_meta_data( '_stripe_customer_id', $customer_id );
+	}
+
+	/**
+	 * Get the Stripe customer ID.
+	 *
+	 * @return string
+	 */
+	public function get_stripe_customer_id() {
+		return $this->get_meta( '_stripe_customer_id' );
+	}
+
+	/**
+	 * Set the charge captured flag.
+	 *
+	 * @param $value string The value to set.
+	 * @return void
+	 */
+	public function set_charge_captured( $value ) {
+		$this->update_meta_data( '_stripe_charge_captured', $value );
+	}
+	/**
+	 * Whether the charge has been captured.
+	 *
+	 * @return bool
+	 */
+	public function charge_captured() {
+		return wc_string_to_bool( $this->get_meta( '_stripe_charge_captured' ) );
+	}
+
+	/**
+	 * Set the status final value.
+	 *
+	 * @param $value bool The value to set.
+	 * @return void
+	 */
+	public function set_status_final( $value ) {
+		$this->update_meta_data( '_stripe_status_final', $value );
+	}
+
+	/**
+	 * Whether the current order status is final.
+	 *
+	 * @return bool
+	 */
+	public function status_final() {
```

Comment:
For consistency, this should be `is_status_final` or `get_status_final`

Reply:
Addressed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3654/commits/1dc95df6d048882b482e40171c6af42b7ce32796